stdlib/2/__builtin__.pyi

```diff
@@ -1283,6 +1283,8 @@ if sys.version_info >= (3,):
     @overload
     def max(_arg1: _T, _arg2: _T, *_args: _T, key: Callable[[_T], Any] = ...) -> _T: ...
     @overload
+    def max(_iterable: Iterable[_T], *, key: Callable[[_T], Any] = ...,) -> _T: ...
+    @overload
     def max(_iterable: Iterable[_T], *, key: Callable[[_T], Any] = ..., default: _VT = ...) -> Union[_T, _VT]: ...
```

Comment:
Won't mypy still complain about calls without `default=` because this overload can still be called that way? (I.e. why not make `default` mandatory in this overload?

Reply:
mypy will take the first overload, I think, so the change did fix the issue, but I will make it mandatory